[Portuguese] Translate guide/README.md (#889)
grammyjs/website: site/docs/pt/guide/README.md

![grammY](/images/grammY.webp)

# Visão Geral

Boas vindas ao grammY!
Você veio para o lugar certo.

## O Que É o grammY?

O grammY é um framework para criar bots do telegram.
Ele pode ser usado com TypeScript e JavaScript e roda em Node.js, Deno, e no navegador.

Dê uma olhada na [Introdução](./introduction) se você está começando no mundo dos bots do Telegram, especialmente se você tem pouca experiência com programação.

Se você sabe como escrever uma aplicação em Node.js ou Deno ou já criou algum bot do Telegram com outro framework para bots, comece em poucos minutos com o nosso guia de [Primeiros Passos](./getting-started).

> ## Sobre a documentação

Reviewer: ```suggestion
## Sobre a Documentação
```

A documentação do grammY é dividida em três camadas.

1. Documentação do grammY de alto nível _(este site)_
2. [Referência da API de baixo nível do grammY](https://deno.land/x/grammy/mod.ts)
3. [Referência da API HTTP bruta do Telegram](https://core.telegram.org/bots/api)

**A primeira parte** (você está olhando pra ela!) explica como os bots funcionam e como usar o grammY.
É o que você vai ver com mais frequência.
A seção _Aprenda_ é sempre um bom start, mas também vale a pena conferir nossa grande coleção de _Plugins_, e dar uma olhada nos _Exemplos_.

**A segunda parte** é a [referência de API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
É uma visão detalhada de cada pedacinho de código que o grammY fornece.

> É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis das dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.

Reviewer: ```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

What about this? Remove the word entirely?

Author: perfect!

**A terceira parte** é fornecida pelo Telegram e lista as definições brutas da [API HTTP](https://core.telegram.org/bots/api) à qual o grammY vai se conectar por debaixo dos panos.
A referência de API do grammY aponta pra lá sempre que faz sentido.
Dê uma olhada lá quando estiver interessado em opções detalhadas que você pode passar para chamadas de API.

> ::: tip: Entre na Comunidade!

Author: ```suggestion
::: tip Entre na Comunidade!
```

![grammY](/images/grammY.webp)

# Visão Geral

Bem-vindo ao grammY!
Você veio para o lugar certo.

## O Que É o grammY?

O grammY é um framework para criar bots do telegram.
Ele pode ser usado com TypeScript e JavaScript e roda em Node.js, Deno, e no navegador.

Dê uma olhada na [Introdução](./introduction) se você é novo no mundo dos bots do Telegram, especialmente se você tem pouca experiência com programação.

Se você sabe como escrever uma aplicação em Node.js ou Deno ou já criou algum bot do Telegram com outro framework para bots, comece em poucos minutos com o nosso guia de [Primeiros Passos](./getting-started).

## Sobre a documentação

A documentação do grammY é dividida em três camadas.

> 1. Documentação do grammY de alto nível _(this website)_

Reviewer: ```suggestion
1. Documentação do grammY de alto nível _(este site)_
```

2. [Referência da API de baixo nível do grammY](https://deno.land/x/grammy/mod.ts)
3. [Referência da API HTTP bruta do Telegram](https://core.telegram.org/bots/api)

**A primeira parte** (você está olhando pra ela!) explica como os bots funcionam e como usar o grammY.
É o que você vai ver com mais frequência.

> A seção _Aprenda_ é sempre um bom start.
> Mas também passe pela nossa grande coleção de _Plugins_, e dê uma olhada nos _Exemplos_.

Reviewer: ```suggestion
A seção _Aprenda_ é sempre um bom start, mas também vale a pena conferir nossa grande coleção de _Plugins_, e dar uma olhada nos _Exemplos_.
```

> **A segunda parte** é a  [referência da API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.

Author: ```suggestion
**A segunda parte** é a [referência da API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
```

> 
> **A segunda parte** é a [referência da API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
> É uma visão geral bem detalhada de cada pedacinho de código que o grammY fornece.

Reviewer: ```suggestion

**A segunda parte** é a [referência de API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
É uma visão detalhada de cada pedacinho de código que o grammY fornece.
```

Although "detailed overview" works in English, "visão geral" is basically the opposite of "detalhado", so it doesn't really work the same in Portuguese

> É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis das dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.

Reviewer: ```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis de dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

Idk, this still feels weird. Tbf I've never seen tooltip translated like this 🤔

Author: yeah, I've spent a few minutes trying to find the best fit for it. Still weird. We could flag that to think about later.

Author: ```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis de descrições de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

maybe like that? Or rephrase the entire sentence to find a better arrangement of words with the same meaning?

> **A terceira parte** é fornecida pelo Telegram e lista as definições brutas da [API HTTP](https://core.telegram.org/bots/api) que o grammY vai se conectar debaixo dos panos.

Reviewer: ```suggestion
**A terceira parte** é fornecida pelo Telegram e lista as definições brutas da [API HTTP](https://core.telegram.org/bots/api) à qual o grammY vai se conectar por debaixo dos panos.
```

> A referência da API do grammY aponta pra lá sempre que faz sentido.

Reviewer: ```suggestion
A referência de API do grammY aponta pra lá sempre que faz sentido.
```

Dê uma olhada lá quando estiver interessado em opções detalhadas que você pode passar para chamadas de API.

> ::: dica: Entre na Comunidade!

Reviewer: ```suggestion
::: tip: Entre na Comunidade!
```

We should not translate these prefixes, as they are actually special markdown code that generates the colored portions on the website

> Nós temos um amigável [chat da comunidade](https://t.me/grammyjs) no Telegram que é bem receptivo com novos membros. (Você pode achar o chat russo [aqui](https://t.me/grammyjs_ru).)

Reviewer: ```suggestion
Nós temos um [chat da comunidade](https://t.me/grammyjs) amigável no Telegram que é bem receptivo com novos membros. (Você pode encontrar o chat russo [aqui](https://t.me/grammyjs_ru).)
```

> Junte-se a nós para receber assistência, fazer perguntas e aprender dicas e macetes para o seu próximo projeto com bot.

Reviewer: ```suggestion
Junte-se a nós para receber ajuda, fazer perguntas e aprender dicas e macetes para o seu próximo projeto de bot.
```
